test/spec/unit/pbjs_api_spec.js

```diff
@@ -2128,14 +2128,47 @@ describe('Unit: Prebid Module', function () {
         $$PREBID_GLOBAL$$.requestBids({adUnits});
         const spyArgs = adapterManager.callBids.getCall(0);
         const nativeRequest = spyArgs.args[1][0].bids[0].nativeParams;
-        expect(nativeRequest).to.deep.equal({
-          image: {required: true},
-          title: {required: true},
-          sponsoredBy: {required: true},
-          clickUrl: {required: true},
-          body: {required: false},
-          icon: {required: false},
-        });
+        expect(nativeRequest.ortb.assets).to.deep.equal([
```

Comment:
Wouldn't we want to have a test case of old native request as well? It should be able to accept either, if ortb is set go with that if not use old, right? Maybe it's too much to ask or it is already covered in another test.

Reply:
It's already covered in another test.  
Basically all existing native tests rely on 'old native way'.  
But, if you feel like this PR needs more tests, I'll add them?

Reply:
If it's covered in another test, then that works!